packages/sdk/src/modules/storage/index.ts

```diff
@@ -0,0 +1,248 @@
+import { ApiPromise } from '@polkadot/api';
+import type { ISubmittableResult } from '@polkadot/types/types';
+import type { CodecHash } from '@polkadot/types/interfaces/runtime/types';
+import { CreateStorageKeysEnum, Address } from '../../types';
+import { createStorageKeys } from '../../utils';
+import { stringToU8a, u8aToHex, hexToString } from '@polkadot/util';
+import { StorageError, ItemTypeError, ItemError, StorageAddressError, StorageSeedError} from '../../utils/errors';
+import type { SDKMetadata} from '../../types';
+import { Base } from '../base';
+
+type AddItemOptions = {
```

Comment:
currently have it so only a string can be set for itemType and item. Should this be changed to offer more types?